ui/app/registration/controllers/createPatientController.js

```diff
@@ -51,9 +51,32 @@ angular.module('bahmni.registration')
                     }).value();
                 };
 
-                _.chain(personAttributeTypes)
+                var isDateTypeAndDefaultValueToday = function (personAttributeType) {
+                    return isDateType(personAttributeType) && isDefaultValueToday(personAttributeType);
+                };
+
+                var isDateType = function (personAttributeType) {
+                    return personAttributeType.format === "org.openmrs.util.AttributableDate";
+                };
+
+                var isDefaultValueToday = function (personAttributeType) {
+                    if (defaults[personAttributeType.name].toLowerCase() === "today") {
+                        return true;
+                    }
+                    $scope.patient[personAttributeType.name] = '';
```

Comment:
@sowmika148 Instead of setting empty value in boolean method, we can set it in 'setDefaultTodayDate' method which we have separately to set data to the patient. Please refactor accordingly.